components/globals/footer/TheFooter.vue

```diff
@@ -37,7 +243,221 @@ const handleUpdateCookieConsent = () => {
 
 <style lang="postcss" scoped>
 .footer {
-  background-color: var(--color-primary);
-  padding: 80px 0 60px @(--sm) 90px 0 90px;
+  background-color: var(--theme-background-color);
+  color: var(--theme-color);
+  padding: 72px 0 @(--sm) 96px 0;
+  transition: background-color 0.3s ease-in-out;
+  position: relative;
+
+  &__section {
+    z-index: 1;
+  }
+
+  &__title {
+    margin-bottom: 0;
+    padding: 0;
+    font-size: 28px;
+    color: var(--theme-footer-text-color);
+  }
+
+  &__contact-info {
+    padding-top: 20px;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: var(--theme-footer-text-color);
+
+    :deep(p):last-child {
+      padding-bottom: 0;
+      margin-bottom: 0;
+    }
+  }
+
+  &__contact-info-button {
+    padding-top: 20px;
+  }
+
+  &__contact-button {
+    color: var(--theme-footer-text-color);
+    border-bottom: 2px solid var(--theme-footer-text-color);
+
+    &:hover {
+      border-bottom: 2px solid var(--theme-button-ghost-text-color-hover);
+
+      :deep(path) {
+        stroke: var(--theme-button-ghost-text-color-hover);
+      }
+    }
+  }
+
+  &__button-icon {
+    padding-left: 12px;
+    font-size: 24px;
+
+    :deep(path) {
+      transition: stroke 0.3s ease-in-out;
+      stroke: var(--theme-footer-text-color);
+    }
+  }
+
+  &__contact-social-media {
+    padding-top: 48px @(--sm) 36px;
+    font-size: 32px;
+    display: flex;
+    gap: 24px;
+
+    &--desktop {
+      display: none @(--md) flex;
+    }
+
+    &--mobile {
+      display: flex @(--md) none;
+    }
+  }
+
+  &__contact-social-media-icon {
+    text-decoration: none;
+    cursor: pointer;
+
+    :deep(path) {
+      fill: var(--theme-footer-text-color);
+    }
+
+    :deep(rect) {
+      stroke: var(--theme-footer-text-color);
+    }
+
+    :deep(path),
+    :deep(rect) {
+      transition: all 0.3s ease-in-out;
+    }
+
+    &:hover {
+      :deep(path) {
+        fill: var(--theme-button-ghost-text-color-hover);
+      }
+      :deep(rect) {
+        stroke: var(--theme-button-ghost-text-color-hover);
+      }
+    }
+  }
+
+  &__shortcuts {
+    padding-top: 48px @(--sm) 0;
+    margin-bottom: 0 @(--md) 20px;
+    display: grid;
+  }
+
+  &__shortcuts-container {
+    padding-top: 20px;
+    display: grid;
+  }
+
+  &__shortcuts-item {
+    padding-top: 8px;
```

Comment:
Remove the padding-top and add it as margin-bottom instead.
Because you have padding-top on .footer__shortcuts-container which matches the .footer__contact-info padding-top but with the padding-top on each shortcut item you get a misalignment between the two footer columns from the title to the content below of 8px.